packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -217,6 +217,53 @@ If your `Choice` doesn't have an `otherwise()` and none of the conditions match
+* `Condition.stringLessThanEquals` - Matches if a string field sorts equal to or before a given value
+* `Condition.stringLessThanEqualsJsonPath` - Matches if a string field sorts equal to or before a given mapping
+* `Condition.stringGreaterThan` - Matches if a string field sorts after a given value
+* `Condition.stringGreaterThanJsonPath` - Matches if a string field sorts after a value at a given mapping path
+* `Condition.stringGreaterThanEqualsJsonPath` - Matches if a string field sorts after or equal to value at a given mapping path
+* `Condition.stringGreaterThanEquals` - Matches if a string field sorts after or equal to a given value
+* `Condition.numberEquals` - Matches if a numeric field has the given value
+* `Condition.numberEqualsJsonPath` - Matches if a numeric field has the value in a given mapping path
+* `Condition.numberLessThan` - Matches if a numeric field is less than the given value
+* `Condition.numberLessThanJsonPath` - Matches if a numeric field is less than the value at the given mapping path
+* `Condition.numberLessThanEquals` - Matches if a numeric field is less than or equal to the given value
+* `Condition.numberLessThanEqualsJsonPath` - Matches if a numeric field is less than or equal to the numeric value at given mapping path
+* `Condition.numberGreaterThan` - Matches if a numeric field is greater than the given value
+* `Condition.numberGreaterThanJsonPath` - Matches if a numeric field is greater than the value at a given mapping path
+* `Condition.numberGreaterThanEquals` - Matches if a numeric field is greater than or equal to the given value
+* `Condition.numberGreaterThanEqualsJsonPath` - Matches if a numeric field is greater than or equal to the value at a given mapping path
+* `Condition.timestampEquals` - Matches if a timestamp field is the same time as the given timestamp
+* `Condition.timestampEqualsJsonPath` - Matches if a timestamp field is the same time as the timestamp at a given mapping path
+* `Condition.timestampLessThan` - Matches if a timestamp field is before the given timestamp
+* `Condition.timestampLessThanJsonPath` - Matches if a timestamp field is before the timestamp at a given mapping path
+* `Condition.timestampLessThanEquals` - Matches if a timestamp field is before or equal to the given timestamp
+* `Condition.timestampLessThanEqualsJsonPath` - Matches if a timestamp field is before or equal to the timestamp at a given mapping path
+* `Condition.timestampGreaterThan` - Matches if a timestamp field is after the timestamp at a given mapping path
+* `Condition.timestampGreaterThanJsonPath` - Matches if a timestamp field is after the timestamp at a given mapping path
+* `Condition.timestampGreaterThanEquals` - Matches if a timestamp field is after or equal to the given timestamp
+* `Condition.timestampGreaterThanEqualsJsonPath` - Matches if a timestamp field is after or equal to the timestamp at a given mapping path
+* `Condition.stringMatches` 
 ### Parallel
```

Comment:
```suggestion
* `Condition.stringMatches` 

### Parallel
```

Reply:
changed existing commit to incorporate this